vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -99,16 +100,16 @@
         @SerializedName(STATUS_QUOTA_EXCEEDED)
         QUOTA_EXCEEDED(STATUS_QUOTA_EXCEEDED);
 
-        private String string;
+        private final String mString;
```

Comment:
Minor: this variable could be a little more descriptive - like "statusTitle" or something